pkgs/applications/blockchains/besu/default.nix

```
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "besu";
-  version = "21.10.8";
+  version = "22.4.3";
```

```suggestion
  version = "22.4.4";
```

A new version is out. Please update.